corehq/apps/cleanup/management/commands/int_id_to_bigint.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from django.apps import apps
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connections
+
+from corehq.form_processor.utils.sql import fetchone_as_namedtuple
+from corehq.util.django_migrations import add_if_not_exists_raw
+from corehq.util.log import with_progress_bar
+from six.moves import input
+
+
+def confirm(table_name):
+    return input(
+        u"""
+        Are you sure you want to modify the {} table? [y/N]
+        """.format(table_name)
```

Comment:
nit: this is going to be kinda ugly because of extra whitespace in the string. Consider adding `.strip()`

Reply:
I actually quite like the way this dispalys on the console.